<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UB7KQ0Q" failures="1" tests="8" name="Test1" time="79.993" errors="0" timestamp="2025-07-10T14:38:18 IST">
  <testcase classname="testcases.LoginTestProctor" name="verifyForgotPassword" time="9.23"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.492">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at testcases.LoginTestProctor.verifyWithInvalidCredentials(LoginTestProctor.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verifyWithInvalidCredentials -->
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.662"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.471"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.418"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithValidCredentials" time="4.299"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyResetPassword" time="36.98"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyResetPassword" time="16.64"/>
</testsuite> <!-- Test1 -->
